src/main/java/com/ibm/nfvodriver/service/LifecycleManagementService.java

```diff
@@ -37,44 +37,44 @@ public LifecycleManagementService(NSLifecycleManagementDriver nsLifecycleManagem
     public ExecutionAcceptedResponse executeLifecycle(ExecutionRequest executionRequest) throws MessageConversionException {
         logger.info("Processing execution request");
 
-        try {
-            if ("Install".equalsIgnoreCase(executionRequest.getLifecycleName())) {
-                // Generate CreateNsRequest message
-                final String createNsRequest = messageConversionService.generateMessageFromRequest("CreateNsRequest", executionRequest);
-
-
-
-                // Send message to NFVO
-                final String nsInstanceResponse = nsLifecycleManagementDriver.createNsInstance(executionRequest.getDeploymentLocation(), createNsRequest);
-
-                // Convert response into properties to be returned to ALM
-                final Map<String, Object> outputs = messageConversionService.extractPropertiesFromMessage("NsInstance", executionRequest, nsInstanceResponse);
-
-                final String requestId = UUID.randomUUID().toString();
-                // Delay sending the asynchronous response (from a different thread) as this method needs to complete first (to send the response back to Brent)
-                externalMessagingService.sendDelayedExecutionAsyncResponse(new ExecutionAsyncResponse(requestId, ExecutionStatus.COMPLETE, null, outputs, Collections.emptyMap()), properties.getExecutionResponseDelay());
+        try {if("Create".equalsIgnoreCase(executionRequest.getLifecycleName())) {
```

Comment:
You may need to format the code to look clean, I see try and if are cluttered in single line.

Reply:
did the format and committed back